app/views/users/edit.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_for @user, method: :patch do |f| %>
+   <% if @user.errors.any? %>
+    <% @user.errors.full_messages.each do |e| %>
+      <%= e%><br>
+    <% end %>
+  <% end %> 
```

Comment:
I understand that you're trying to print error messages if the creation is not successful. 

However, instead of duplicating this code block in individual views (new.html.erb & edit.html.erb), you can simply include it in your application layout file (layouts/application.html.erb). 

This will ensure that the code snippet is available across all views in your app.